extensions/kafka/src/main/java/com/hazelcast/jet/kafka/KafkaProcessors.java

```diff
@@ -145,4 +145,18 @@ public static <T, K, V> ProcessorMetaSupplier writeKafkaP(
     ) {
         return ProcessorMetaSupplier.of(1, WriteKafkaP.supplier(dataConnectionRef, toRecordFn, exactlyOnce));
     }
+
+    /**
+     * Returns a supplier of processors for
+     * {@link KafkaSinks#kafka(Properties, FunctionEx)}.
+     */
+    public static <T, K, V> ProcessorMetaSupplier writeKafkaP(
```

Comment:
I can see that we want to support creation of producers for data connection with and without additional properties.
If so, describe this assumption somewhere (javadoc, jira, PR commit description).